dev/Breadcrumb/Strings/en-us/Resources.resw

```diff
@@ -121,4 +121,8 @@
     <value>More</value>
     <comment>The automation name for the BreadcrumbBar ellipsis button.</comment>
   </data>
+  <data name="BreadcrumbBarItemLocalizedControlType" xml:space="preserve">
+    <value>Breadcrumb Bar Item</value>
```

Comment:
You should only use lowercase characters (see https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.automationproperties.localizedcontroltypeproperty?view=winrt-19041).

Reply:
I took inspiration from the Rating Control, should we file a bug for it?